CPS-?/README.md

```diff
@@ -0,0 +1,170 @@
+
+In rare instances, a threat may be identified that requires responsible reporting procedures to prevent exposing an active vulnerability to bad actors seeking to exploit it. In such cases, it is essential to notify the appropriate group or entity with the ressources, or are responsible for mitigating the threat in a confidential manner.
+
+### #1: Unknown software vulnerability affecting governance is discovered.
+
+An assumption is that are no known software back doors. Unknown software vulnerabilities may be discovered.
+
+Recommendation: The person discovering the vulnerability should document the circumstances and details in which the vulnerability is present and how to reproduce the vulnerability. Then report the vulnerability in a confidential manner using a secure method to the developer team with the resources to correct the vulnerability.
+
+### #2: Emergency group to support the disaster recovery plan.
+  -attacks on persons if known
+  -we don’t want attackers know who that group is?
+
+Recommendation:
+
+### #3: Emergency change that needs to be done quickly.
+  -a nuclear option
+  -change requires “just do it” 
+  -a security that fix must be implemented but CANNOT be communicated publicly
+
+Recommendation:
+
+### #4: Operational Security (OPSEC) of the committee.
+  -physical security
+  -digital security
+
+Recommendation:
+
+### #5: Committee keys lost or stolen or committee keys sold to a bad actor.
+
+Recommendation:
+
+### #6: DRep keys lost or stolen or DRep keys sold to a bad actor.
+
+Recommendation:
+
+### #7: Lack of governance for an extended period of time.
+  -Uncontrolled hard forks by SPOs
+  -Risk of no confidence
+
+Recommendation:
+
+### #8: Lack of trust in voter tooling.
+
+Recommendation:
+
+### #9: Trust in the wallets.
+  -wallet voting attack vector
+  -hardware wallet support
+
+Recommendation:
+
+
+### #10: Corruption - in any form, any number of actors.
+  -Code of Conduct mitigation
+  -on-chain mitigation
+
+
+Recommendation:
+
+### #11: Collusion - the bad version of collaboration to do something illegal or undesirable.
+
+Recommendation:
+
+### #12: Vote buying - Ada kickbacks.
+
+Recommendation:
+
+### #13: IGCO - initial governance coin offering
+  -offering voters a token of unknown value by a representative to acquire delegators
+  -including tokens other than Ada
+
+Recommendation:
+
+### #14: Powerful Voter Collusion
+  -demotivates smaller bag holders
+  -cause a loss of participation
+  -too many voters abstain
+  -minority controls the outcome
+  -can push through any proposal
+
+Recommendation:
+
+### #15: Persistence - Removal of proposals by attackers.
+
+Recommendation:
+
+### #16: Risk of Proposal overload - Risk of DDOS.
+
+Recommendation:
+
```

Comment:
- Require collateral to accompany a proposal (i.e. 50 $Ada or higher)

- Collateral reimbursement once proposal is approved for a voting round or rejected due to constitutional violation

Reply:
Same as above, I think we need to define the difference between a deposit and collateral, and which entity would have the power to slash it. Also might need to consult the game theory experts on slashing mechanisms as a form of punishment.